### ISSUE TITLE
[Bug] VS Code + Robocorp LSP + Robocop: Top-most section throws error

### ISSUE THREAD

**Issue by GLMeece** [MarketSquare/robotframework-robocop#679]
### What happened?

# Summary

Error message:

> `Error collecting Robocop errors: 'Variable' object has no attribute 'error'`

This produces the "red squiggly line" as well as showing errors in the **PROBLEMS** tab of the console.

Editor "squiggly":
![image](https://user-images.githubusercontent.com/7154245/185658114-3581a06f-0134-4780-ab38-bc7a991ce3c6.png)

**PROBLEMS** section in the console:
![image](https://user-images.githubusercontent.com/7154245/185658416-8517da90-527b-492f-b9db-c0454234e24f.png)

Of note:
- You can swap the `*** Settings ***` section with the `*** Variables ***` section as the first line and you'll get the same result.
- You can add a newline to the top of the file, and it'll throw this error on the blank first line.
- In other words, it's always the first line. Everything else seems to work correctly (e.g., if you make a syntax or formatting mistake, it catches it as it should).

## Setup

- Using Robocop + [Robocorp's LSP](https://robocorp.com/docs/developer-tools/visual-studio-code/overview) [in VS Code](https://marketplace.visualstudio.com/items?itemName=robocorp.robotframework-lsp).

### Operating System

Windows 10

### Robocop version

2.3.0

**Comment by GLMeece** [comment]
OK, once I upgraded to the latest version _and_ reloaded the LSP, this issue went away. I had **thought** I had upgraded earlier, but such was not the case.

IMO, this can be closed.
